src/tooltips.tsx

```diff
@@ -152,52 +154,80 @@ class TooltipComponent extends React.Component<{}, TooltipComponentState> {
 		boundingRect.top += window.scrollY;
 		boundingRect.left += window.scrollX;
 
-		const { placement, containerStyle, innerStyle, arrowStyle } = options;
-
-		// Set the contents of the tooltip using `ìnnerText` and adjust the styles
-		// now, so that the height can be properly calculated
-		if (this.tooltipElementInner) {
-			this.tooltipElementInner.innerText = tooltipText;
+		if (!this.tooltipElement) {
```

Comment:
We already check for this, so the additional check is redundant. I think we need to create a repeatable failing test case and then patch around it. This solution feels too much like guesswork.

Reply:
Oh! Nice spot! It seems that I duplicated this during the refactor.

Reply:
The fact is though that I couldn't reproduce what we saw in sandbox under normal circumstances :(